### PR TITLE
Realign the README file

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -414,29 +433,62 @@ To provide a new name, use the same pragma and provide a name as parameter:
...
 
-The returned values of your final tasks will be automatically printed.
+Same behavior of the Kernel to display console outputs. To print a task output, you can use the the 
+`#%print_task_output()` pragma:
+
+```python
+#%print_task_output([job_id=JOB_ID], [job_name=JOB_NAME], task_name=TASK_NAME)
+```
+
+and finally, to print all job outputs, the use may use the `#%print_job_output()` pragma by providing the
```
Comment (luispinedamx) [inline README.md]: the use*r*

File: README.md [hunk 2]
```diff
@@ -414,29 +433,62 @@ To provide a new name, use the same pragma and provide a name as parameter:
...
+To get the result(s) of all the tasks of a job, the user has to use the `#%get_job_result()` pragma by providing the 
+job name or the job ID:
 
 ```python
-#%get_result(id=JOB_ID)
+#%get_job_result([job_id=JOB_ID], [job_name=JOB_NAME])
 ```
 
-The returned values of your final tasks will be automatically printed.
+Same behavior of the Kernel to display console outputs. To print a task output, you can use the the
```
Comment (luispinedamx) [inline README.md]: "the the"

File: README.md [hunk 3]
```diff
@@ -491,18 +543,32 @@ Features:
...
 
 * *get_result*: gets and prints the job results
 
+* *get_job_result*: gets and prints the job results
+
+* *get_task_result*: gets and prints the results of a given task
+
+* *print_job_output*: gets and prints the job outputs
+
+* *print_task_output*: gets and prints the outputs of a given task\n
```
Comment (luispinedamx) [inline README.md]: is there a reason why there is a \n only here and not in the rest of the list elements?

Comment (mklkun) [reply README.md]: Oops! I tried some stuff and I forgot to delete this one. lol

**Pull request merged**